src/templates/pages/home.html

```diff
@@ -31,7 +33,22 @@ <h2>Accelerating reproducible computational research.</h2>
     {#                </div>#}
     {#            </div>#}
     {#        </div>#}
-
+    <script>
+    // Prevent buttons from stacking in mobile view
+    $(document).ready(function () {
+        $(window).resize(function resize() {
+            if ($(window).width() < 390) {
+                return $('.pages')
+                    .removeClass('huge')
+                    .addClass('large');
+            }
+
+            $('.pages')
+                .removeClass('large')
+                .addClass('huge');
+        }).trigger('resize');
```

Comment:
Ahh! That's neat, instead of having to make resize a function and re-using/calling it, `trigger` is cool, not sure I've seen that before..